front/pages/w/[wId]/builder/data-sources/managed.tsx

```diff
@@ -594,238 +657,284 @@ export default function DataSourcesView({
             </Hoverable>
           </ContentMessage>
         )}
-        <ContextItem.List>
-          {localIntegrations
```

Comment:
What's `localIntegrations` here?

Reply:
kept the previous naming. Will rename to `dataSourceIntegration` which is more explicit